Application Lifecycle Management/api-blobs.md

```diff
@@ -0,0 +1,186 @@
+{{{
+"title": "Blobs API",
+"date": "09-01-2016",
+"author": "",
+"attachments": [],
+"contentIsHTML": false
+}}}
+
+**Manage Blobs**
+
+| Resource | Description |
+|----------|-------------|
+| [POST /services/blobs/upload](#post-servicesblobsupload) | Uploads a file using multi-part form data |
+| [POST /services/blobs/upload/{file_name}](#post-servicesblobsuploadfile_name) | Creates a blob from submitted data |
+| [GET /services/blobs/download/{file_id}/{file_name}](#get-servicesblobsdownloadfile_idfile_name) | Downloads a file uploaded previously |
+
+## POST /services/blobs/upload
+Uploads a file using multi-part form data.
+
+### URL
+
+#### Structure
+```
+[POST] /services/blobs/upload
+```
+#### Example
+```
+[POST] https://cam.ctl.io/services/blobs/upload
+```
+### Request
+
+#### Headers
+```
+Content-Type: multipart/form-data
+Authorization: Bearer your_json_web_token
+ElasticBox-Release: 4.0
+```
+#### URI Parameters
+* None
+#### Request body parameters
+* Form data: blob (binary)
+
+### Response
+#### Normal Response Codes
+
+- **200** OK
+
+#### Common Error Response Codes
+
+- **400** Bad Request
+- **401** Unauthorized
+
+#### Response Parameters
+
+| Parameter | Type |Description |
+|-----------|------|------------|
+| content-type | string | File content type |
+| length | number | File size in bytes |
+| upload_date | string | Date when file was uploaded |
+| url | string | Url to download the file |
+
+#### Response Body
+
+```
+{
+    "url": "/services/blobs/download/5be2de281862a32389a50d81/Screen-Shot-2018-11-07-at-13.44.12.png",
+    "upload_date": "2018-11-07 12:44:24.254860",
+    "length": 12715,
+    "content_type": "image/png"
+}
+```
+## POST /services/blobs/upload/{file_name}
+Creates a blob from submitted data.
+
+### URL
+
+#### Structure
+```
+[POST] /services/blobs/upload/{file_name}
+```
+#### Example
+```
+[POST] https://cam.ctl.io/services/blobs/upload/another-file-name.png
+```
+### Request
+
+#### Headers
+```
+Content-Type: multipart/form-data
+Authorization: Bearer your_json_web_token
+ElasticBox-Release: 4.0
+```
+#### URI Parameters
+* None
+#### Request body parameters
+* Form data: blob (binary)
+
```

Comment:
It must be located within the previous section URI Parameters.
Here must be 
* None